src/test/java/Homework23.java

```diff
@@ -0,0 +1,107 @@
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.AllSongsPage;
+import pages.BasePage;
+import pages.HomePage;
+import pages.LoginPage;
+
+
+public class Homework23 extends BasePage
+{
+    public Homework23(WebDriver givenDriver)
+    {
+        super(givenDriver);
+    }
+//    @Test (dataProvider = "CorrectLoginProvider", dataProviderClass = BaseTest.class)
+//    public void deleteAPlaylist(String email, String password)
+//    {
+//        String deletedPlayListMessage = "Deleted playlist";
+//        //accessUrlPage();
+//        provideEmailCredentials("james.patterson@testpro.io");
+//        providePasswordCredentials("te$t$tudent");
+//        loginButton();
+//        openPlaylist();
+//        clickDeletePlaylistButton();
+//        confirmDeletedPlaylistMessage();
+//        Assert.assertTrue(getDeletedPlaylistMessage().contains(deletedPlayListMessage));
+//    }
+
+//    @Test (dataProvider = "CorrectLoginProvider", dataProviderClass = BaseTest.class)
+//    public void createAPlaylist(String email, String password)
+//    {
+//        String createAPlaylist = "Created playlist ";
+//
+//        provideEmailCredentials("james.patterson@testpro.io");
+//        providePasswordCredentials("te$t$tudent");
+//        loginButton();
+//
+//        clickOnPlusSymbol();
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.